bifrost.go

```diff
@@ -191,3 +194,35 @@ func newSimpleStorageService(g *BridgeConfig) (RainbowBridge, error) {
 		Client:        client,
 	}, nil
 }
+
+func newGoogleDriveStoage(g *BridgeConfig) (rainbowBridge, error) {
+	var client *http.Client
```

Comment:
For this http client, I think the `shared/request` package can be used here...most likely...just that it might take some default value parameters...eg `""`